toggl/mobileapp
# Fix overlapping font sizes

### Description

Fixes UI uglyness on larger font/display sizes.

### Relationships

Closes #2668
Ref #2676 

### What has been done in this PR?

* Renamed from `Item` to `TimeEntry` (as these indeed are only time entries)
* UI looks better on larger font/display sizes
    * Main View: Suggestions and Time Entries
    * Start TE View: Time entry auto suggestions
    * Edit TE View: Duration text should now be fully visible

### How to validate?

Please open those views with font/display sizes set to different sizes. Also, make sure to test possible combinations of time entries (no description, no project, etc.)

### 🦑 permissions
Everyone.
